src/main/java/server/request/RequestParser.java

```diff
@@ -0,0 +1,33 @@
+package server.request;
+
+public class RequestParser {
+  private String request;
+
+  public Request parse(String request) {
+    this.request = request;
+
+    return new Request(method(), path(), protocol(), body());
+  }
+
+  private String startLine() {
+    return request.split("\r\n")[0];
+  }
+
+  private String method() {
+    return startLine().split(" ")[0];
+  }
+
+  private String path() {
+    return startLine().split(" ")[1];
+  }
+
+  private String protocol() {
+    return startLine().split(" ")[2];
+  }
+
+  private String body() {
```

Comment:
How is the body going to look like if you receive the body with multiple lines in request? Do you have tests for that?

Reply:
I didn't test for this bc it never occurred to me! Will look into it now, thanks!